cmd/root/auth.go

```diff
@@ -107,7 +105,6 @@ TRY_AUTH: // or try picking a config profile dynamically
 	if err != nil {
 		return err
 	}
-	ctx = context.WithValue(ctx, &currentUser, me)
```

Comment:
Is there anything that depends on this?

Reply:
There is not. The only accessor was the `root.Me()` function and nothing calls it.